### PR TITLE
First round of broken link replacment/removal

### PR DESCRIPTION
**Pull request by johnwhitington**

### CODE REVIEW

File: site/community/support.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ OCaml community, with projects such as
 OCaml to JavaScript optimizing compiler),
 [Dose](https://gforge.inria.fr/projects/dose/) (a key component of the
 OPAM package manager) and online services such as the [OPAM Weather
-Service](http://ows.irill.org/latest/today/). They also use OCaml
+Service](http://check.ocamllabs.io). They also use OCaml
```
Comment (avsm) [inline site/community/support.md]: Just removing this link is appropriate.  The check service is pretty different from the opam weather service. It's fine to just unlink it I think.

**Review by avsm — APPROVED** [review]
Thanks for the very comprehensive set of changes. Just one minor change, but otherwise good to merge.

**Comment by johnwhitington** [comment]
Thanks!

By the way, do you have an OCaml contact at ahrefs? One of things they do is a broken links checker, and I wondered if we could sweet-talk them into giving us a free account...

**Comment by avsm** [comment]
@Khady also offered this a few days ago. Hooking that up would be much appreciated

**Comment by Khady** [comment]
I've run a crawl of ocaml.org (limited to ocaml.org itself, subdomains such as opam.ocaml.org were not crawled), which covered 6,743 pages. 66 of them have broken links.
 
[ocaml-org-07-Sep-2020.zip](https://github.com/ocaml/ocaml.org/files/5181111/ocaml-org-07-Sep-2020.zip)

You can ignore the first column. It is not relevant for what is done here.

|Page rating (desc)|URL                                                                                         |Internal outlinks to 4xx                                                                                                                                                                                                                    |Internal outlinks to 5xx|External outlinks to 4xx                                                                                                                                                                                                                                                                                           |External outlinks to 5xx                        |
|------------------|--------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------|
|69                |https://ocaml.org/releases/4.00/htmlman/index.html                                          |https://ocaml.org/releases/4.00/ocaml-4.00-refman.html.tar.gz                                                                                                                                                                               |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|67                |https://ocaml.org/releases/4.01/htmlman/index.html                                          |https://ocaml.org/releases/4.01/ocaml-4.01-refman.html.tar.gz                                                                                                                                                                               |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|62                |https://ocaml.org/community/planet/                                                         |                                                                                                                                                                                                                                            |                        |http://ocamllabs.io/feed.xml                                                                                                                                                                                                                                                                                       |https://examachine.net/blog/category/ocaml/feed/|
|61                |https://ocaml.org/learn/tutorials/                                                          |                                                                                                                                                                                                                                            |                        |http://www.linux-nantes.org/~fmonnier/OCaml/ocaml-wrapping-c.html                                                                                                                                                                                                                                                  |                                                |
|60                |https://ocaml.org/docs/install.html                                                         |                                                                                                                                                                                                                                            |                        |https://lists.fedoraproject.org/mailman/listinfo/Ocaml-devel                                                                                                                                                                                                                                                       |                                                |
|60                |https://ocaml.org/community/mailing_lists.html                                              |                                                                                                                                                                                                                                            |                        |https://vector.im/beta/                                                                                                                                                                                                                                                                                            |                                                |
|37                |https://ocaml.org/releases/4.06/htmlman/libref/Arg_helper.html                              |https://ocaml.org/releases/4.06/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.06/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|37                |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.html                       |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.Make.Map.html https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.Make.Set.html https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.Make.Tbl.html|                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|37                |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.html                       |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.Make.Map.html https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.Make.Set.html https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.Make.Tbl.html|                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|37                |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.html                       |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.Make.Map.html https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.Make.Set.html https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.Make.Tbl.html|                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|37                |https://ocaml.org/releases/4.07/htmlman/libref/Arg_helper.html                              |https://ocaml.org/releases/4.07/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.07/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|36                |https://ocaml.org/releases/4.05/htmlman/libref/Arg_helper.html                              |https://ocaml.org/releases/4.05/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.05/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|36                |https://ocaml.org/releases/4.03/htmlman/libref/Arg_helper.html                              |https://ocaml.org/releases/4.03/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.03/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|36                |https://ocaml.org/releases/4.04/htmlman/libref/Arg_helper.html                              |https://ocaml.org/releases/4.04/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.04/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|35                |https://ocaml.org/releases/4.08/htmlman/compilerlibref/Arg_helper.html                      |https://ocaml.org/releases/4.08/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.08/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|35                |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Arg_helper.html                      |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.09/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|35                |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Arg_helper.html                      |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.10/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|35                |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Arg_helper.html                      |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.11/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|34                |https://ocaml.org/docs/install.fr.html                                                      |                                                                                                                                                                                                                                            |                        |https://lists.fedoraproject.org/mailman/listinfo/Ocaml-devel                                                                                                                                                                                                                                                       |                                                |
|34                |https://ocaml.org/meetings/ocaml/2015/                                                      |                                                                                                                                                                                                                                            |                        |http://www.mlworkshop.org/ml2015/                                                                                                                                                                                                                                                                                  |                                                |
|33                |https://ocaml.org/meetings/ocaml/2016/                                                      |                                                                                                                                                                                                                                            |                        |http://opam.ocamlpro.com/builder/html/report-last.html http://www.mlworkshop.org/ml2016/                                                                                                                                                                                                                           |                                                |
|33                |https://ocaml.org/learn/tutorials/ocamlbuild/                                               |                                                                                                                                                                                                                                            |                        |http://gallium.inria.fr/~pouillar/ocamlbuild/html/Signatures.PLUGIN.html http://gallium.inria.fr/~pouillar/ocamlbuild/ocamlbuild-user-guide.html                                                                                                                                                                   |                                                |
|33                |https://ocaml.org/learn/tutorials/index.ja.html                                             |                                                                                                                                                                                                                                            |                        |http://www.linux-nantes.org/~fmonnier/OCaml/ocaml-wrapping-c.html                                                                                                                                                                                                                                                  |                                                |
|31                |https://ocaml.org/learn/companies.html                                                      |                                                                                                                                                                                                                                            |                        |http://www.prudentconsulting.com/                                                                                                                                                                                                                                                                                  |                                                |
|29                |https://ocaml.org/community/cwn/                                                            |https://ocaml.org/community/cwn/mailing_lists.html                                                                                                                                                                                          |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.Make.html                  |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.Make.Map.html https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.Make.Set.html https://ocaml.org/releases/4.09/htmlman/compilerlibref/Consistbl.Make.Tbl.html|                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.Make.html                  |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.Make.Map.html https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.Make.Set.html https://ocaml.org/releases/4.10/htmlman/compilerlibref/Consistbl.Make.Tbl.html|                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.Make.html                  |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.Make.Map.html https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.Make.Set.html https://ocaml.org/releases/4.11/htmlman/compilerlibref/Consistbl.Make.Tbl.html|                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/meetings/ocaml/2017/                                                      |                                                                                                                                                                                                                                            |                        |http://www.mlworkshop.org/ml2017                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/learn/tutorials/compiling_ocaml_projects.html                             |https://ocaml.org/learn/tutorials/dev_tools.md                                                                                                                                                                                              |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.06/htmlman/libref/Arg_helper.Make.html                         |https://ocaml.org/releases/4.06/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.06/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.07/htmlman/libref/Arg_helper.Make.html                         |https://ocaml.org/releases/4.07/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.07/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.03/htmlman/libref/Arg_helper.Make.html                         |https://ocaml.org/releases/4.03/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.03/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.05/htmlman/libref/Arg_helper.Make.html                         |https://ocaml.org/releases/4.05/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.05/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.04/htmlman/libref/Arg_helper.Make.html                         |https://ocaml.org/releases/4.04/htmlman/libref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.04/htmlman/libref/Arg_helper.Make.Value.html                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.08/htmlman/compilerlibref/Arg_helper.Make.html                 |https://ocaml.org/releases/4.08/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.08/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Arg_helper.Make.html                 |https://ocaml.org/releases/4.09/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.09/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|29                |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Arg_helper.Make.html                 |https://ocaml.org/releases/4.10/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.10/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|28                |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Arg_helper.Make.html                 |https://ocaml.org/releases/4.11/htmlman/compilerlibref/Arg_helper.Make.Key.html https://ocaml.org/releases/4.11/htmlman/compilerlibref/Arg_helper.Make.Value.html                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|28                |https://ocaml.org/releases/4.08.0.html                                                      |https://ocaml.org/releases/4.08/ocaml-4.08-refman.ps.gz                                                                                                                                                                                     |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|28                |https://ocaml.org/releases/4.01.0.html                                                      |                                                                                                                                                                                                                                            |                        |https://admin.fedoraproject.org/pkgdb/package/ocaml/                                                                                                                                                                                                                                                               |                                                |
|28                |https://ocaml.org/releases/4.00.1.html                                                      |                                                                                                                                                                                                                                            |                        |https://admin.fedoraproject.org/pkgdb/package/ocaml/                                                                                                                                                                                                                                                               |                                                |
|28                |https://ocaml.org/releases/3.12.1.html                                                      |                                                                                                                                                                                                                                            |                        |http://rpm.nogin.org/ocaml.html                                                                                                                                                                                                                                                                                    |                                                |
|28                |https://ocaml.org/learn/tutorials/basics.zh.html                                            |                                                                                                                                                                                                                                            |                        |https://ocsigen.org/js_of_ocaml/2.7/files/toplevel/index.html                                                                                                                                                                                                                                                      |                                                |
|28                |https://ocaml.org/meetings/ocaml/2009/                                                      |                                                                                                                                                                                                                                            |                        |http://home.gna.org/cameleon/chamo.en.html                                                                                                                                                                                                                                                                         |                                                |
|28                |https://ocaml.org/meetings/ocaml/2008/                                                      |                                                                                                                                                                                                                                            |                        |http://gallium.inria.fr/~pouillar/ocamlbuild/ocamlbuild-ocamlmeeting08.pdf                                                                                                                                                                                                                                         |                                                |
|28                |https://ocaml.org/learn/tutorials/basics.ko.html                                            |                                                                                                                                                                                                                                            |                        |https://ocsigen.org/js_of_ocaml/2.7/files/toplevel/index.html                                                                                                                                                                                                                                                      |                                                |
|28                |https://ocaml.org/learn/tutorials/basics.el.html                                            |                                                                                                                                                                                                                                            |                        |https://ocsigen.org/js_of_ocaml/2.7/files/toplevel/index.html                                                                                                                                                                                                                                                      |                                                |
|28                |https://ocaml.org/ocamllabs/                                                                |                                                                                                                                                                                                                                            |                        |http://gow.epsrc.ac.uk/NGBOViewGrant.aspx?GrantRef=EP/G065802/1                                                                                                                                                                                                                                                    |                                                |
|28                |https://ocaml.org/learn/libraries.html                                                      |                                                                                                                                                                                                                                            |                        |https://github.com/languages/OCaml                                                                                                                                                                                                                                                                                 |http://caml.inria.fr/cgi-bin/hump.en.cgi        |
|27                |https://ocaml.org/learn/teaching-ocaml.html                                                 |                                                                                                                                                                                                                                            |                        |http://www.cs.umd.edu/class/spring2014/cmsc330/ http://www.cs.bc.edu/~muller/teaching/cs110105/f14/ http://www.cs.virginia.edu/~weimer/4610/ https://international.uni.wroc.pl/en/course/functional-programming https://services.brics.dk/java/courseadmin/dOvs https://realworldocaml.org/v1/en/html/prologue.html|                                                |
|27                |https://ocaml.org/meetings/ocaml/2012/                                                      |https://ocaml.org/meetings/ocaml/2012/cfp.txt                                                                                                                                                                                               |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|27                |https://ocaml.org/learn/tutorials/ocamlbuild/Making_plugins.html                            |                                                                                                                                                                                                                                            |                        |http://gallium.inria.fr/~pouillar/ocamlbuild/html/Signatures.COMMAND.html http://gallium.inria.fr/~pouillar/ocamlbuild/html/Signatures.OPTIONS.html http://gallium.inria.fr/~pouillar/ocamlbuild/html/Signatures.OUTCOME.html http://gallium.inria.fr/~pouillar/ocamlbuild/html/Signatures.PLUGIN.html             |                                                |
|27                |https://ocaml.org/caml-light/                                                               |https://ocaml.org/caml-light/releases/index.html                                                                                                                                                                                            |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|27                |https://ocaml.org/learn/tutorials/ocamlbuild/Using_an_external_library.html                 |                                                                                                                                                                                                                                            |                        |http://gallium.inria.fr/~pouillar/ocamlbuild/html/Signatures.PLUGIN.html                                                                                                                                                                                                                                           |                                                |
|27                |https://ocaml.org/learn/tutorials/garbage_collection.zh.html                                |https://ocaml.org/img/users.bin.gz                                                                                                                                                                                                          |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|27                |https://ocaml.org/learn/history.html                                                        |                                                                                                                                                                                                                                            |                        |http://www-sop.inria.fr/members/Manuel.Serrano/ http://homepages.inf.ed.ac.uk/rburstall/                                                                                                                                                                                                                           |                                                |
|27                |https://ocaml.org/learn/tutorials/garbage_collection.ja.html                                |https://ocaml.org/img/users.bin.gz                                                                                                                                                                                                          |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|27                |https://ocaml.org/learn/tutorials/file_manipulation.zh.html                                 |https://ocaml.org/releases/4.11/htmlman/libref/Pervasives.html                                                                                                                                                                              |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|27                |https://ocaml.org/learn/tutorials/file_manipulation.ja.html                                 |https://ocaml.org/releases/4.11/htmlman/libref/Pervasives.html                                                                                                                                                                              |                        |http://ocaml-extlib.googlecode.com/svn/doc/apiref/Std.html                                                                                                                                                                                                                                                         |                                                |
|26                |https://ocaml.org/docs/consortium-license.html                                              |https://ocaml.org/docs/consortium-license.fr.html                                                                                                                                                                                           |                        |                                                                                                                                                                                                                                                                                                                   |                                                |
|26                |https://ocaml.org/learn/tutorials/ocamlbuild/Using_an_ocamlfind_installation_of_sexplib.html|                                                                                                                                                                                                                                            |                        |http://www.ocaml.info/home/ocaml_sources.html                                                                                                                                                                                                                                                                      |                                                |
|26                |https://ocaml.org/community/planet/older.html                                               |                                                                                                                                                                                                                                            |                        |http://gallium.inria.fr/blog/how-i-found-a-bug-in-intel-skylake-processord http://gallium.inria.fr/blog/mirage-retreat-field-trip-report http://gallium.inria.fr/blog/parser-construction-with-menhir-a-couple-appetizers                                                                                          |                                                |
|26                |https://ocaml.org/learn/tutorials/camlp4_3.10/foreach_tutorial.html                         |                                                                                                                                                                                                                                            |                        |http://brion.inria.fr/gallium/index.php/Using_Camlp4                                                                                                                                                                                                                                                               |                                                |
|26                |https://ocaml.org/meetings/ocaml/2014/tom-ridge-slides.html                                 |                                                                                                                                                                                                                                            |                        |http://www.tom-ridge.com/doc/ridge14sle_extended.pdf                                                                                                                                                                                                                                                               |                                                |
|26                |https://ocaml.org/learn/history.fr.html                                                     |                                                                                                                                                                                                                                            |                        |http://pauillac.inria.fr/coq/index.fr.html http://www-sop.inria.fr/members/Manuel.Serrano/                                                                                                                                                                                                                         |                                                |

**Comment by Khady** [comment]
@johnwhitington we do give free accounts to ocaml related organizations. I can probably arrange that. Just need to know who would manage the account.

**Comment by johnwhitington** [comment]
@Khady Thanks, on both counts. I am happy to manage the account, but I wonder if ocaml.org has a policy on where these sorts of login details reside - perhaps @avsm could comment?

**Comment by johnwhitington** [comment]
I have fixed the rest of the broken external links per the ahrefs report. I will leave the internal ones for another day, so this can be merged if you are happy.

**Comment by Chris00** [comment]
Thanks for your work!

**Pull request merged**